Clarify content exclusion support in Copilot modes

queuedecay opened the pull request:
Updated note on content exclusion support in various modes. The article https://github.com/github/docs/blob/main/content/copilot/how-tos/configure-content-exclusion/exclude-content-from-copilot.md contains a better note. Copying the note from that page. The current text in this article could mislead users into thinking the feature will work in editors other than VS Code.

<!--
Thank you for contributing to this project! You must fill out the information below before we can review this pull request. By explaining why you're making a change (or linking to an issue) and what changes you've made, we can triage your pull request to the best possible team for review.
-->

### Why:

<!-- Paste the issue link or number here -->
Closes: https://github.com/github/docs/issues/40951

<!-- If there's an existing issue for your change, please link to it above.
If there's _not_ an existing issue, please open one first to make it more likely that this update will be accepted: https://github.com/github/docs/issues/new/choose. -->

### What's being changed (if available, include any code snippets, screenshots, or gifs):

<!-- Let us know what you are changing. Share anything that could provide the most context.
If you made changes to the `content` directory, a table will populate in a comment below with links to the review and current production articles. -->

I simply copied line 34 from https://github.com/github/docs/blob/main/content/copilot/how-tos/configure-content-exclusion/exclude-content-from-copilot.md, and then pasted it over line 51 in this document. FYI, I'm a Microsoft GBB, not GitHub staff.

### Check off the following:

- [ ] A subject matter expert (SME) has reviewed the technical accuracy of the content in this PR. In most cases, the author can be the SME. Open source contributions may require an SME review from GitHub staff.
- [ ] The changes in this PR meet [the docs fundamentals that are required for all content](http://docs.github.com/en/contributing/writing-for-github-docs/about-githubs-documentation-fundamentals).
- [ ] All CI checks are passing and the changes look good in the review environment.

queuedecay committed:
Clarify content exclusion support in Copilot modes

Updated note on content exclusion support in various modes. The article https://github.com/github/docs/blob/main/content/copilot/how-tos/configure-content-exclusion/exclude-content-from-copilot.md contains a better note. Copying the note from that page. The current text in this article could mislead users into thinking the feature will work in editors other than VS Code.

Sharra-writes committed:
Merge branch 'main' into patch-1

Sharra-writes commented:
@queuedecay Thanks for bringing this up! My main issue with this edit is that it's giving information that has no apparent relevance to the subject, at least from my perspective (keep in mind that I'm not an engineer). One principle of good writing is that every word you add detracts and distracts from every other word already there. My preference would be to change the note to say, "Content exclusion is currently not supported in Edit and Agent modes of Copilot Chat in Visual Studio Code and other IDEs," because it stays on topic and adds very few additional words.

That's a writer's perspective. If you have a relevant engineering perspective that's different, I'm definitely willing to be persuaded.

queuedecay committed:
Merge branch 'main' into patch-1

queuedecay committed:
Clarify content exclusion support in Copilot modes

Shortened changes to simply add "and other editors" to line 53, which removes the confusion in as few words as possible.

queuedecay commented:
Wonderful recommendation. Considered, approved, and applied, after fetching the current docs state.

Sharra-writes commented:
@queuedecay "Editor" should be fine. It looks like we use it pretty interchangeably with "IDE" in the docs. We're on a deploy freeze for GitHub Universe, but I can get this merged after that's lifted.

Sharra-writes committed:
Merge branch 'main' into patch-1
